### PR TITLE
Add results for 9.8 compare test for 3 of the runs

### PR DESCRIPTION
**Pull request by DHaussermann**
Add results for 9.8 compare test for 3 of the runs

MySlq Unbounded is running for a 2nd time. I will add results shortly.

### CODE REVIEW

File: performance-comparisons/v9.8/results/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Mattermost load-test-ng
```
Comment (agarciamontoro) [inline performance-comparisons/v9.8/results/README.md:1]: I think this whole file got added inadvertently! We should remove it from this PR.

**Review by agarciamontoro — COMMENTED** [review]
This looks great! I'll wait until the latest test finishes and results are up to get it merged! Btw, are you planning to add the `results.txt` file when that one is finished?

The only thing to fix is that there's a couple of files that shouldn't be on this PR. Both seem to have been added inadvertently:
- `results/README.md`: this seems to be the original README from the load-test repo
- `graphs/.DS_Store`: macOS just doing its usual thing :joy:

**Comment by DHaussermann** [comment]
@agarciamontoro Yes I will add results file once I have all 4 blocks current. 
Should be within a couple hours. 
- For `results/README.md` Sorry this was a silly oversight. I copied across `*.md` files and did not realize I got an unwanted one.
Yes, I will remove the Mac OS system file as well :)

**Comment by DHaussermann** [comment]
@agarciamontoro please review again to see if all looks correct. 
Note that in the results block and the report files we have 2 run `0`s this time as the MySql Unbounded results were generated separately so they are also number `0`.

File: README.md [hunk 2]
```diff
@@ -1,7 +0,0 @@
-# Performance Reports
```
Comment (agarciamontoro) [inline README.md:1]: You did delete the other copied README, but it seems you have also deleted the original one!

File: performance-comparisons/v9.8/config/comparison.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "BaseBuild": {
+"Label": "release-9.7.0",
+"URL": "https://releases.mattermost.com/9.7.0/mattermost-enterprise-9.7.0-linux-amd64.tar.gz"
+  },
+  "NewBuild": {
+"Label": "release-9.8.0-rc2",
+"URL": "https://releases.mattermost.com/9.8.0-rc2/mattermost-enterprise-9.8.0-rc2-linux-amd64.tar.gz"
+  },
```
Comment (agarciamontoro) [inline performance-comparisons/v9.8/config/comparison.json:9]: Can we indent this?
```suggestion
{
  "BaseBuild": {
  "Label": "release-9.7.0",
  "URL": "https://releases.mattermost.com/9.7.0/mattermost-enterprise-9.7.0-linux-amd64.tar.gz"
  },
  "NewBuild": {
  "Label": "release-9.8.0-rc2",
  "URL": "https://releases.mattermost.com/9.8.0-rc2/mattermost-enterprise-9.8.0-rc2-linux-amd64.tar.gz"
  },
```

File: performance-comparisons/v9.8/config/deployer.json [hunk 4]
```diff
@@ -0,0 +1,113 @@
+{
+"ClusterName": "dkhcomparison",
```
Comment (agarciamontoro) [inline performance-comparisons/v9.8/config/deployer.json:2]: Same indentation problem here
```suggestion
  "ClusterName": "dkhcomparison",
```

Comment (DHaussermann) [reply performance-comparisons/v9.8/config/deployer.json]: Resolved with latest commit.

**Comment by DHaussermann** [comment]
Thanks @agarciamontoro for taking the time on this. Can you please review again to ensure that I understood and implemented the changes correctly?

**Comment by agarciamontoro** [comment]
> Thanks @agarciamontoro for taking the time on this. Can you please review again to ensure that I understood and implemented the changes correctly?

All good, thanks!

> Also I need help to get rid of performance-comparisons/v9.8/graphs/.DS_Store I see it in the commit in the GitHub UI but, it's some type of untracked change so the IDE does not show it so I don't know how to commit a delete for it. Also the option in the GutHub UI to delete is greyed out. Thoughts?

Mmmm, I don't know how to do it from the IDE, but from a terminal, you need something like:

```sh
git rm --cached v9.8/graphs/.DS_Store
git commit -m "Remove .DS_Store file"
```

The `--cached` flag in `git rm` is important to only remove the file from git, but not from the filesystem. This SO answer has the details: https://stackoverflow.com/a/2047477

**Review by agarciamontoro — APPROVED** [review]
Neat, thanks!

**Pull request merged**